crates/analytics/src/auth_events/metrics/authentication_count.rs

```diff
@@ -52,23 +51,7 @@ where
         }
 
         query_builder
-            .add_filter_clause("merchant_id", publishable_key)
-            .switch()?;
-
-        query_builder
-            .add_bool_filter_clause("first_event", 1)
-            .switch()?;
-
-        query_builder
-            .add_filter_clause("category", "USER_EVENT")
-            .switch()?;
-
-        query_builder
-            .add_filter_clause("log_type", "INFO")
-            .switch()?;
-
-        query_builder
-            .add_filter_clause("event_name", SdkEventNames::ThreeDsMethod)
+            .add_filter_clause("merchant_id", _merchant_id)
```

Comment:
```suggestion
            .add_filter_clause("merchant_id", merchant_id)
```
rename function argument as well